tests/fixtures/tree-sitter-lint-plugin-replace-foo-with-bar/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
```

Comment:
I tried putting this under `examples/` but it seems like those can only be "standalone" `.rs` files (vs entire Cargo-based projects)

So I just put it under a new `tests/fixtures/` directory assuming that we will probably want some stuff like this for integration tests